packages/components/src/textarea-control/README.md

```diff
@@ -1,63 +1,137 @@
 # TextareaControl
 
-TextareaControl is used to generate textarea input fields.
+TextareaControls are TextControls that allow for multiple lines of text, and wrap overflow text onto a new line. They are a fixed height and scroll vertically when the cursor reaches the bottom of the field.
 
+![An empty TextareaControl, and a focused TextareaControl with some content entered.](https://wordpress.org/gutenberg/files/2019/01/TextareaControl.png)
 
-## Usage
+## Table of contents
 
-```jsx
-import { TextareaControl } from '@wordpress/components';
-import { withState } from '@wordpress/compose';
+1. [Design guidelines](http://#design-guidelines)
+2. [Development guidelines](http://#development-guidelines)
+3. [Related components](http://#related-components)
 
-const MyTextareaControl = withState( {
-	text: '',
-} )( ( { text, setState } ) => ( 
-	<TextareaControl
-		label="Text"
-		help="Enter some text"
-		value={ text }
-		onChange={ ( text ) => setState( { text } ) }
-	/>
-) );
-```
+## Design guidelines
 
-## Props
+### Usage
+
+**When to use TextareaControl**
+
+Use TextareaControl when you need to encourage users enter an amount of text that’s longer than a single line. (A bigger box can encourage people to be more verbose, where a smaller one encourages them to be succinct.)
+
+TextareaControl should:
+
+- Stand out from the background of the page and indicate that users can input information.
+- Have clearly differentiated active/inactive states, including focus styling.
+- Make it easy to understand and address any errors via clear and direct error notices.
+- Make it easy to understand the requested information by using a clear and descriptive label.
+
+**When not to use TextareaControl**
+
+Do not use TextareaControl if you need to let users enter shorter answers no longer than a single line, such as a phone number or name. In this case, you should use `Text Control`.
```

Comment:
This is a little confusing to read for me: "...shorter answers no longer than a single line". Maybe it could use a comma after "shorter answers"?

Reply:
Updated to:

> Do not use TextareaControl if you need to let users enter shorter answers (no longer than a single line), such as a phone number or name. In this case, you should use `Text Control`.

Reply:
Perfect! Thanks